README.md

```diff
@@ -118,7 +118,7 @@ Imagine that you take the code from this question, save it to a file called `nam
 
 What would you have to type to exit the while loop?  Explain your answer.
 
-<!-- Replace this comment with your answer -->
+you'd have to write "SeCrEt". you know this because in line 5, the code specifies that while the answer is not '' and is not 'SeCrEt', the code will continue to ask "Guess my secret?"
```

Comment:
Yep! Though, as is mentioned in the condition, typing nothing (empty string) will also work.